operator/Makefile

```
@@ -3,7 +3,12 @@
 APP_NAME = keda-manager
 IMG_REPO := $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)
 IMG_NAME := $(IMG_REPO)/$(APP_NAME)
+
+# accept environmental variables
+ifeq ($(IMG),)
 IMG := $(IMG_NAME):$(DOCKER_TAG)
+endif
```

I think this code would do the same but it's more readable

```suggestion
IMG ?= $(IMG_NAME):$(DOCKER_TAG)
```

---

here is explanation 
https://stackoverflow.com/questions/24777289/what-is-in-makefile